lib/web_ui/lib/src/engine/window.dart

```diff
@@ -564,7 +536,7 @@ class EngineWindow extends ui.Window {
         final MethodCall decoded = codec.decodeMethodCall(data);
         switch (decoded.method) {
           case 'SystemNavigator.pop':
-            _browserHistory.exit().then((_) {
+            browserHistory.exit().then((_) {
```

Comment:
This is a getter for test, we should use _browserHistory

Reply:
I changed it so that it does lazy initialization of `_browserHistory`.